department-of-veterans-affairs/caseflow
# Div/appeals 42572

[Correspondence and Appeal Tasks: Inbound Ops Team Superuser/visor: Remove Evidence Submission Window](https://jira.devops.va.gov/browse/APPEALS-42572)

# Description
Implemented the feature as mentioned in the AC

## Acceptance Criteria

## Testing Plan
1. Sign in as an INBOUND OPS TEAM ADMIN
2. Navigate to Queue
3. Click the **Switch Views** dropdown and select **correspondence cases**.
4. Select a correspondence from the **unassigned** tab.
5. On the Review Package page, select a correspondence type from the dropdown (if one doesn't already exist) and click **save**.
6. Click **establish** at the bottom to create a new intake.
7. Click **continue** through the first page of correspondence intake.
8. On the second page, click the radio button under **existing appeals** section. Select 2 appeals that have the `EvidenceWindowSubmissionTask` showing. 
9. For the first appeal, waive the task and provide notes. For the last task, do not waive the task (this will be the control). 
10. Click **continue**.
11. On the 3rd step, copy the URL minus the `/intake`. Click **establish** to complete the intake.
12. Using the URL from step 11 or by finding the correspondence in the **completed** tab of the **correspondence cases** page, navigate to the Correspondence Details page. 
13. Expectation is that the waived evidence task shows an action dropdown "**Remove waive evidence window action** " to un-waive, and the other does not show any task.
14. From the modal, you can click confirm to un-waive.
15. Once you click confirm, the Waivable task action is expected to be gone and the success banner will appear. Also it updates related Existing Appeal's grey table to have the new task.



# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
